.github/workflows/comment-on-pull-request.yml

```diff
@@ -0,0 +1,16 @@
+name: comment-on-pull-request
+on: pull_request
+permissions: 
+  pull-requests: write 
```

Comment:
Is this needed?

Reply:
I included this after looking at the documentation of the GitHub Action used from the market place (https://github.com/marketplace/actions/comment-pull-request#permissions) to resolve the `Resource not accessible by integration
` error (which still has not been resolved even after adding it). But if this can be solved directly by changing the configs on the repo's settings, then I think we can remove it.

Reply:
@eddiejaoude can you help us here?